services/lemmy/lemmy.service.js

```diff
@@ -0,0 +1,90 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService, InvalidParameter } from '../index.js'
+
+const queryParamSchema = Joi.object({
+  server_fqdn: Joi.string().hostname(),
+}).required()
+
+const lemmyCommunitySchema = Joi.object({
+  community_view: Joi.object({
+    counts: Joi.object({
+      subscribers: Joi.number().required(),
+      posts: Joi.number().required(),
+      comments: Joi.number().required(),
```

Comment:
If the only value we're going to use is subscribers we can drop posts/comments from the schema